test/ufuzz.js

```diff
@@ -202,7 +191,11 @@ var ASSIGNMENTS = [
     '<<=',
     '>>=',
     '>>>=',
-    '%=' ];
+    '%=',
+];
+ASSIGNMENTS = ASSIGNMENTS.concat(ASSIGNMENTS.map(function() {
+    return '=';
```

Comment:
I appreciate feedbacks like these, since I'm the kind of person who never understand the need for obfuscation software, as they seem to be doing worse... :ghost: